### DISCUSSION TITLE
Blind Level Action Stop Trigger

### DISCUSSION THREAD

**Discussion by R4ygin** [Zefau/ioBroker.jarvis#935] [Q&A]
Hallo,

ich versuche derzeit Rolladen von Bosch Smart Home einzubinden. Dabei habe ich die zwei Datenpunkte level und activity eingerichtet. Der Datenpunkt level funktioniert soweit und ist mit einem BlindLevelAction Button auf der Startseite verknüpft, hat als primären Datentyp level und sekundären Datentyp activity. Jetzt möchte ich aber mit einem Klick auf Stop dem activity den Wert "STOPPED" zuweisen. Ich habe bemerkt, dass der Klick dem activity den Wert "true" zuweist, ich habe jedoch noch nicht herausgefunden wie ich dieses "true" zu einem "STOPPED" umwandeln kann. Kann mir diesbezüglich jemand helfen?

Hier der Code:
```JSON
"censoredId": {
    "id": "censoredId",
    "function": "blind",
    "states": {
      "level": {
        "state": {
          "node": "bshb.0.hdm:HomeMaticIP:censoredId.ShutterControl.level"
        },
        "action": "bshb.0.hdm:HomeMaticIP:censoredId.ShutterControl.level",
        "bodyElement": "LevelBody",
        "unit": {
          "0": null,
          "1": null,
          "default": "%"
        },
        "properties": {
          "min": 0,
          "max": 1
        },
        "display": {
          "0": "offen",
          "1": "geschlossen"
        },
        "LevelBodyConfig": {
          "step": "0.01"
        },
        "icon": {
          "default": "window-shutter-open",
          ">=0.5": "window-shutter-open",
          "<0.5": "window-shutter"
        },
        "stateStyle": "",
        "actionElement": "BlindLevelAction",
        "hide": false
      },
      "activity": {
        "state": {
          "node": "bshb.0.hdm:HomeMaticIP:censoredId.ShutterControl.operationState"
        },
        "action": "bshb.0.hdm:HomeMaticIP:censoredId.ShutterControl.operationState",
        "display": {
          "MOVING": "in Bewegung",
          "STOPPED": "gestoppt"
        },
        "icon": {
          "MOVING": "pan-vertical",
          "STOPPED": "dots-vertical"
        },
        "properties": {
          "off": "STOPPED",
          "on": "MOVING"
        },
        "actionElement": "SwitchAction",
        "stateStyle": {
          "MOVING": {
            "color": "#0D2"
          }
        }
      }
    },
    "options": {},
    "attributes": {
      "_created": "censoredNumber",
      "_updated": "censoredNumber"
    },
    "name": "Wohnzimmer Rollade Fenster",
    "hash": "censoredNumber"
  }
```

Vielen Dank im Voraus.

LG Ray

**Comment by testit2020** [comment]
Hast du da mal nachgeschaut?
https://zefau.gitbook.io/jarvis-de/geraete/functions/blind
Man muss einen eigenen DP für STOP haben.
oder hier: 
https://forum.iobroker.net/topic/30668/jarvis-v2-1-0-just-another-remarkable-vis/3568

https://forum.iobroker.net/topic/30668/jarvis-v2-1-0-just-another-remarkable-vis/3577

**Comment by RkcCorian** [comment]
Hab auch Bosch und hab's Folgendermaßen gelöst: Datenpunkt "activity" (Ist wohl in JARIVS hardcodiert) angelegt, dieser Triggered  einen ALIAS mit folgenden Inhalt:
    "alias": {
      "id": "bshb.0.hdm:HomeMaticIP:3014F711A00018DA1859B70C.ShutterControl.operationState",
      "write": "val='STOPPED'"
    }

**Comment by testit2020** [comment]
@RkcCorian kannst du mal bitte das Gerät als Bild darstellen? Wo wird das mit dem alias eingetragen? Danke.

**Comment by RkcCorian** [comment]
Ich nutze den Adapter "Manage devices", wobei ich inzw. die Alias manuell anlege. Statt Alias kannst Du natürlich auch einen eigenen Datenpunkt anlegen und den Aufruf über Script machen.

![image](https://user-images.githubusercontent.com/79741077/112802972-1ac9be80-9073-11eb-9766-269149b2f840.png)
![image](https://user-images.githubusercontent.com/79741077/112803070-3339d900-9073-11eb-89d5-80bb965f61d5.png)
![image](https://user-images.githubusercontent.com/79741077/112803145-464ca900-9073-11eb-8e0f-12799c7e6109.png)
